tests/cupyx_tests/scipy_tests/ndimage_tests/test_interpolation.py

```diff
@@ -55,6 +55,13 @@ def test_map_coordinates_float(self, xp, scp, dtype):
         coordinates = testing.shaped_random((a.ndim, 100), xp, dtype)
         return self._map_coordinates(xp, scp, a, coordinates)
 
+    @testing.for_float_dtypes(no_float16=True)
+    @testing.numpy_cupy_allclose(atol=1e-5, scipy_name='scp')
+    def test_map_coordinates_float_nd_coords(self, xp, scp, dtype):
+        a = testing.shaped_random((100, 100), xp, dtype)
+        coordinates = testing.shaped_random((a.ndim, ) + a.shape, xp, dtype)
```

Comment:
- `a`'s shape and `coordinates`' shape essentially have no relation except `a.ndim` and 10,000 points are, I suppose, more than enough for testing here.
- How about supplying `scale=99.0` to cover `a`'s coordinates?
```suggestion
        coordinates = testing.shaped_random((a.ndim, 10, 10), xp, dtype, scale=99.0)
```